packages/bson-transpilers/test/yaml/partial.yaml

```diff
@@ -179,6 +193,7 @@ tests:
           java: Pattern
           csharp: Regex
           shell: RegExp
+          php: ''
```

Comment:
I'm noting that you chose to leave _this_ example empty, but `native.yaml` used strings. I'm not sure why they're different if PHP doesn't have a native regex type.

Reply:
I chose to use string-regexps but in this case this string has no constructing (operating) class. The internal presentation lets you convert the common regexp from the input into some specific class with constructing argument. In our case we have no constructing class, just only the argument which is a common string. That is why for this test we have an empty output: we don't have an operating regexp class.

Reply:
OK to resolve.